key-rotator/key/key.go

```diff
@@ -73,6 +74,65 @@ func (k Key) Equal(o Key) bool {
 	return true
 }
 
+// Diff returns a human-readable string describing the differences from the
+// given `o` key to this key, suitable for logging. Diff returns the empty
+// string if and only if the two keys are equal.
+func (k Key) Diff(o Key) string {
+	// Build up structures allowing easy generation of diffs.
+	var newPKTS, oldPKTS *int64
```

Comment:
It took me a minute but I eventually figured out that `PKTS` expands to `PrimaryKeyTimeStamp`. Since the type is `int64` and not something like `time.Time` that makes it more obvious, and since `PK` could also easily mean "public key" or "private key", could we expand the acronym for readability?